docs/mcp_guide.md

```diff
@@ -0,0 +1,232 @@
+- `deadline_list_storage_profiles_for_queue()`: List storage profiles for a queue
+
+**From `deadline.client.api._get_apis`:**
+- `deadline_get_farm()`: Get detailed farm information
+
+**From `deadline.client.api._session`:**
+- `deadline_check_authentication_status()`: Check current authentication status
+
+
+### Job Management Tools
+
+**From `deadline.mcp.tools.job`:**
+- `deadline_submit_job()`: Submit an Open Job Description job bundle to AWS Deadline Cloud
+- `deadline_download_job_output()`: Download job output files from AWS Deadline Cloud
+
+
+### Error Handling and Serialization
+All tools include:
+- Automatic JSON serialization with fallback for complex objects
+- Consistent error response format with error type and message
+- Logging of errors for debugging
+- Graceful handling of missing parameters
+
+### Download Tool Behavior
+The `deadline_download_job_output` tool:
+- Downloads files to their original paths by default
+- Automatically handles AWS credentials and queue permissions
+- Supports conflict resolution for existing files
+- Can download from entire jobs, specific steps, or individual tasks
+
+### Testing the AWS Deadline Cloud MCP Server
+
+## Installation Steps
+
+1. **Install the AWS Deadline Cloud client with MCP support:**
+   ```bash
+   pip install 'deadline[mcp]'
+   ```
+
+2. **Verify installation:**
+   ```bash
+   deadline --version
+   ```
+
+3. **Configure AWS credentials:**
+   - Standard AWS credentials (AWS Profiles, environment variables)
+   - Or Deadline Cloud monitor credentials:
+     ```bash
+     deadline auth login
+     ```
+   - Verify authentication status:
+     ```bash
+     deadline auth status
+     ```
+
+## Starting the MCP Server 
+
+MCP clients can be configured to start the AWS Deadline Cloud MCP Server using the Deadline CLI
+```bash
+deadline mcp-server
+```
+
+### For development and testing
+```bash
+deadline mcp-server
+```
+
+## Integration with MCP Clients
+
+Add to your MCP configuration (e.g., `./settings/mcp.json`):
+```json
+{
+  "mcpServers": {
+    "deadline-cloud": {
+      "command": "deadline",
+      "args": ["mcp-server"],
+      "disabled": false,
+      "autoApprove": []
+    }
+  }
+}
+```
+
+## Example Natural Language Interactions
+
+Once connected to an MCP-enabled AI assistant, you can use natural language to interact with Deadline Cloud. The AI understands context and can help you accomplish complex workflows:
+
+### Farm Management
+```
+"List all my AWS Deadline Cloud farms"
+"Show me the queues in my farm"
+"List the jobs in my queue"
+"Check my authentication status"
+```
+
+### Job Management
+```
+"Submit the render job in /path/to/my-job-bundle"
+"Submit a job with priority 80 to my render queue"
+"Show me the status of job job-3a907bac684841f69fc344867ee166de"
+"Download output from job job-3a907bac684841f69fc344867ee166de"
+"Download output from step step-render in job job-3a907bac684841f69fc344867ee166de"
+```
```

Comment:
Bullet points might be better. Should also not be split up into job and farm management since that's not a meaningful user-facing boundary. I'd guess users are more likely to ask something like, "Why isn't my job being picked up?" which requires both farm and job info to answer.